drop-transfer/src/ws/server/mod.rs

```diff
@@ -1087,6 +1087,14 @@ fn move_tmp_to_dst(
                 continue;
             }
             Err(err) => {
+                #[cfg(windows)]
+                {
+                    // On Win the permissions error is returned in case there's a directory with the
+                    // same name
+                    if path.exists() {
+                        continue;
+                    }
+                }
```

Comment:
Wondering why it doesn't trigger 
```
            Err(err) if err.kind() == io::ErrorKind::AlreadyExists => {
                continue;
            }
```
for Windows. It would be nice not to have alternate paths :thinking:

Reply:
Well, that's what the syscall returns. It returns `AlreadyExists` if there's a conflict with the file but if the conflict is with the folder, it's OS error 5.

Reply:
Well,  I presume that's the question for Microsoft. We can unify the path for all OSes, it should be no harm (I think)

Reply:
Can we do that? It makes for less mental overhead when reasoning about issues of course if possible. The way I see it, it should be no harm to have this check for all OSes not differentiating windows

Reply:
Done

Reply:
nice!